### PR TITLE
Style tweaks

### PR DESCRIPTION
**Pull request by markstory**
* Don't use ID selectors.
* Make sidebar equal height to content region. On long pages they were not equal.
* Add pagination arrows. They might be backwards though, I can never remember which way they are supposed to go.

Related to cakephp/bake#118

### CODE REVIEW

File: webroot/css/cake.css [hunk 1]
```diff
@@ -521,34 +528,47 @@ div.message:before {
...
-#content h3 {
+/* Use 'one true layout' methods to get equal height columns */
+.container {
+  overflow: hidden;
+  min-height: 92%; /* full height almost always */
+}
+
+/* Force equal height by overflowing */
+.content,
+#actions-sidebar {
```
Comment (ADmad) [inline webroot/css/cake.css]: I see an id instead of class :)

Comment (markstory) [inline webroot/css/cake.css]: I only removed `#content` in this pass.

**Comment by lorenzo** [comment]
Good job!

**Comment by mtkocak** [comment]
did you test the bake generator?

**Comment by markstory** [comment]
@mtkocak I did, is there a problem?

**Comment by mtkocak** [comment]
No that's perfect.

2015-06-11 18:56 GMT+02:00 Mark Story <notifications@github.com>:

**Pull request merged**